### PR TITLE
Feature/aruba

### PR DESCRIPTION
**Pull request by FilipDolnik**

### CODE REVIEW

File: Source/GenerateMocksCommand.swift [hunk 1]
```diff
@@ -19,9 +19,10 @@ public struct GenerateMocksCommand: CommandType {
     
     public func run(options: Options) -> Result<Void, CuckooGeneratorError> {
         let parsedFiles = options.files.map { File(path: $0) }.flatMap { $0 }.map { options.runtime.tokenizer.init(sourceFile: $0).tokenize() }
-      let headers = parsedFiles.map { f in options.runtime.fileHeaderHandler.getHeader(options.testableFrameworks, file: f) }
+        let headers = parsedFiles.map { f in options.noHeader ? [] : options.runtime.fileHeaderHandler.getHeader(f) }
+        let imports = parsedFiles.map { f in options.runtime.fileHeaderHandler.getImports(options.testableFrameworks) }
         let mocks = parsedFiles.map(options.runtime.generator.generate)
-        let mergedFiles = zip(headers, mocks).map { $0 + $1 }.map { $0.joinWithSeparator("\n") }
+        let mergedFiles = zip(zip(headers, imports), mocks).map { $0.0 + $0.1 + $1 }.map { $0.joinWithSeparator("\n") }
```
Comment (TadeasKriz) [inline Source/GenerateMocksCommand.swift:25]: Can it be `$0 + $1 + $2` ?

Comment (FilipDolnik) [inline Source/GenerateMocksCommand.swift:25]: No, type of $0 is ([String], [String]).

File: Tests/features/CuckooGeneratorTests.feature [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Feature: CuckooGeneratorTests
+	Scenario: Generating test mocks
+		Given I run `xcodebuild -project '../../../CuckooGenerator.xcodeproj' -scheme 'CuckooGenerator' -configuration 'Release' CONFIGURATION_BUILD_DIR=Tests/tmp/aruba clean build`
```
Comment (TadeasKriz) [inline Tests/features/CuckooGeneratorTests.feature]: This is really too verbose and should not be part of the test. Cuckoo should be already built before testing starts. We definitely don't want to build it in each scenario.

File: Tests/features/CuckooGeneratorTests.feature [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+		When I run `tmp/aruba/cuckoo.app/Contents/MacOS/cuckoo generate --no-header --output actual.swift ../../res/source/TestedClass.swift ../../res/source/TestedProtocol.swift`
+		Then the file "../../res/expected.swift" should be equal to file "actual.swift"
```
Comment (TadeasKriz) [inline Tests/features/CuckooGeneratorTests.feature]: The paths in the test are too verbose as well, we need to make it more easy and pleasant to write those test. There will have to be a lot of them.

File: Tests/features/CuckooGeneratorTests.feature [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Feature: CuckooGeneratorTests
+	Scenario: Generating test mocks
+		When I run `runcuckoo generate --no-header --output Actual.swift ../../Tests/SourceFiles/TestedClass.swift ../../Tests/SourceFiles/TestedProtocol.swift`
```
Comment (TadeasKriz) [inline Tests/features/CuckooGeneratorTests.feature]: Missing given?

**Pull request merged**